.editorconfig

```diff
@@ -16,6 +16,6 @@ indent_size=4
 indent_style=space
 
 [*.{sh,yml,yaml}]
-indent_size=2
+indent_size=4
 indent_style=space
-tab_width=8
+tab_width=4
```

Comment:
Can this be reset to 2 (which conforms to the community convention)?

Reply:
reset to 2?!

the origin code is `tab_width=8`